code/game/objects/items/devices/radio/intercom.dm

```diff
@@ -161,4 +161,14 @@
 
 /obj/item/device/radio/intercom/locked/confessional
 	name = "confessional intercom"
-	locked_frequency = 1480
\ No newline at end of file
+	locked_frequency = 1480
+	
+/obj/item/device/radio/intercom/locked/desk_phone
+	icon = "icons
```

Comment:
the fuck

Reply:
Uh... I don't know why that's like that...